### PR TITLE
Got prusto building with nightly-2021-04-18

### PR DESCRIPTION
**Pull request by bitemyapp**
It wanted the `min_type_alias_impl_trait` feature enabled.

### CODE REVIEW

**Comment by nooberfsh** [comment]
@bitemyapp why is this necessary?

File: rust-toolchain [hunk 1]
```diff
@@ -0,0 +1 @@
+nightly-2021-04-18
```
Comment (nooberfsh) [inline rust-toolchain:1]: Why is this necessary

Comment (bitemyapp) [reply rust-toolchain:1]: It isn't strictly speaking but it does tell people what a known-working compiler version is. I consider this more important when something is nightly-only. YMMV.

You can remove it before merge or I can. Whatever you want.
